File: ruby/vristo-ruby-main/app/controllers/dashboard_controller.rb
class DashboardController < ApplicationController
  def index
    render layout: "default", :template => 'index'
  end
  def analytics
    render layout: "default", :template => 'analytics'
  end
  def finance
    render layout: "default", :template => 'finance'
  end
  def crypto
    render layout: "default", :template => 'crypto'
  end
end

File: symfony/vristo-symfony-starter/src/Controller/DashboardController.php
<?php

namespace App\Controller;

use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
use Symfony\Component\HttpFoundation\Response;
use Symfony\Component\Routing\Annotation\Route;

class DashboardController extends AbstractController
{
    public function index(): Response
    {
        return $this->render('index.html.twig');
    }

    public function analytics(): Response
    {
        return $this->render('analytics.html.twig');
    }

    public function finance(): Response
    {
        return $this->render('finance.html.twig');
    }

    public function crypto(): Response
    {
        return $this->render('crypto.html.twig');
    }

}
